python/test/unit/fem/test_expression.py

```diff
@@ -309,3 +309,72 @@ def gradf(x):
     for i, cell in enumerate(closest_cell):
         point = points[local_map[i]]
         assert np.allclose(grad_u_at_x[i, i], gradf(point))
+
+
+def test_zero_elimination():
```

Comment:
Looks like this test should be in FFCx.

Reply:
I'm not certain on how to easily check this in FFCx (as I am not too familiar with the expression interface there). Maybe @michalhabera has an idea of how to nicely test this.

Reply:
I've now moved the scalar and vector test to FFCx. However, I am reluctant to move the tensor test, as it is rather involved (using Nedelec elements, making creation of the coefficient array rather involved).